README.md

```diff
@@ -1,14 +1,13 @@
-# 🚧 Under construction 🚧
+# ⚠️ Archived ⚠️
 
-WARNING: The Taipy team is performing a repository restructuration. This current repository taipy-config is about to be 
-merged into the main repository: taipy. Once the merge is done, the current code base will be in the 
-[taipy repository](https://github.com/Avaiga/taipy). The migration should take a maximum of a few days.
-<br>
+WARNING: This repository is archived.
+Taipy has performed a repository restructuring. The taipy-config code has been moved and  
+merged into the main [taipy repository](https://github.com/Avaiga/taipy).
 
+<br>
 
-# Taipy config
 
-## License
+# License
 Copyright 2023 Avaiga Private Limited
```

Comment:
So we don't update the date hu? Now would be a good time though

Reply:
Yes, indeed. Let me handle the change then.